aeon/distances/tests/test_pairwise.py

```diff
@@ -99,29 +100,36 @@ def _validate_single_to_multiple_result(
     for i in range(single_to_multiple_result.shape[-1]):
         curr_y = y[i]
         curr = single_to_multiple_result[0, i]
-        dist = distance(x, curr_y)
+
+        curr_x = x
+        if curr_x.ndim > curr_y.ndim:
+            curr_y = curr_y.reshape((1, curr_y.shape[0]))
+        elif curr_x.ndim < curr_y.ndim:
+            curr_x = curr_x.reshape((1, curr_x.shape[0]))
+
+        dist = distance(curr_x, curr_y)
         assert_almost_equal(dist, curr)
 
 
 @pytest.mark.parametrize("dist", DISTANCES)
 def test_pairwise_distance(dist):
     """Test pairwise distance function."""
     _validate_pairwise_result(
-        _create_test_distance_numpy(5, 5),
+        make_series(5, 5, return_numpy=True, random_state=1),
         dist["name"],
         dist["distance"],
         dist["pairwise_distance"],
     )
 
     _validate_pairwise_result(
-        _create_test_distance_numpy(5, 1, 5),
+        make_example_3d_numpy(5, 1, 5, random_state=1)[0],
```

Comment:
I find this more explicit:
```suggestion
        make_example_3d_numpy(5, 1, 5, random_state=1, return_y=False),
```

Reply:
I agree with @CodeLionX :)

Reply:
So this maybe a crazy reason for doing this: The type hint for the function indicated it returns a tuple always and it was setting off my linter. I will just the type hint of the function as well and add return_y=False.